sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/KeyAsyncClientTest.java

```diff
@@ -205,11 +208,17 @@ public void getKey(HttpClient httpClient, KeyServiceVersion serviceVersion) {
 
         getKeyRunner((keyToSetAndGet) -> {
             StepVerifier.create(keyAsyncClient.createKey(keyToSetAndGet))
-                .assertNext(response -> assertKeyEquals(keyToSetAndGet, response))
+                .assertNext(createdKey -> {
+                    assertKeyEquals(keyToSetAndGet, createdKey);
+                    assertNotNull(createdKey.getProperties().getHsmPlatform());
```

Comment:
should we have a check for the actual value also ?

Reply:
See my other comment about this.